include/pacmod_game_control/globals.h

```diff
@@ -127,4 +128,4 @@ static std::mutex turn_mutex;
 }  // namespace Joystick
 }  // namespace AS
 
-#endif  // PACMOD_GAME_CONTROL_GLOBALS_H
+#endif  // PACMOD_GAME_CONTROL_GLOBALS_H
```

Comment:
Do you need to change some settings in your editor? It seems to be removing newline characters at the ends of the files.